fabric/values.yaml

```
@@ -144,17 +144,16 @@ control:
         type: 'null'
         value: '/app/routes.yaml'
     resources:
-      {}
       # We usually recommend not to specify default resources and to leave this as a conscious
       # choice for the user. This also increases chances charts run on environments with little
       # resources, such as Minikube. If you do want to specify resources, uncomment the following
       # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-      # limits:
-      #   cpu: 100m
-      #   memory: 128Mi
-      # requests:
-      #   cpu: 100m
-      #   memory: 128Mi
+      limits:
+        cpu: 250m
```

Let's give this some room to grow than than hard capping it.
```suggestion
        cpu: 500m
```